### DISCUSSION TITLE
Thermaltake fans with plugin being overridden

### DISCUSSION THREAD

**Discussion by mhomorody** [Rem0o/FanControl.Releases#2455] [Q&A] [answered]
I got Fan Control setup with the Thermaltake plugin so I could use these fans https://www.thermaltake.com/pure-plus-12-rgb-radiator-fan-tt-premium-edition-3-fan-pack.html with their hardware controller. I've previously used this TTController software since the official software is bad https://github.com/MoshiMoshi0/TTController but I've never really been happy with the setup. I uninstalled that and all other smart fan controls in my MSI B450-A Pro BIOS are disabled (that's only controlling the CPU fan and Sys Fan #2 which is my bottom intake).

I get the "overridden by an external source" message on these 3 TT fans. It's still there after I force apply. I made a CPU Temp Graph curve that the CPU and Sys Fan respond too, but the percentages on the TT fans stay low or sort of bounce around randomly but low when running Cinebench CPU test. I noticed that the Calibration values for the 3 TT fans are quite different and random as well.

![image](https://github.com/Rem0o/FanControl.Releases/assets/25761436/4b47ced7-b754-4366-8326-b3da32ff4ac1)
![image](https://github.com/Rem0o/FanControl.Releases/assets/25761436/f4ab98db-279a-466a-a95d-98bbe58bcb69)
![image](https://github.com/Rem0o/FanControl.Releases/assets/25761436/dd5a0ebd-e838-4c08-aeb8-d5eac7a6b1b8)

Here's the overall view currently.
![image](https://github.com/Rem0o/FanControl.Releases/assets/25761436/4309ab2c-9a0d-44e7-9b9c-305948594348)

**Comment by Rem0o** [comment]
I don't know the specifics of that controller, but maybe there is a minimum speed it won't accept under.  If you increase the CPU temp graph to like 30%, do the ? signs come off?

**Reply by mhomorody** [reply]
I just went to check that and since I've been installing a bunch of games the CPU was hot and the % was up to 37% already and the ? weren't there anymore. The TT fans were matching the CPU Curve. When running those stress tests or playing a game though I haven't seen the TT fans syncing up. Maybe there's a huge lag, but since I've been updating/installing lots of games for probably an hour, they caught up?

**Reply by mhomorody** [reply]
Watching it more now as more games are installing and the CPU curve is at 42.7% and the TT fans will be at that too, but drop back down to maybe 39% with the ? and then jump back up to stabilize.

edit: I see on the github page for the plugin that it says "Setting the fan power is instant, but the read out takes a few seconds to adjust. So it takes a while before the Fan Control app sees the right RPM and Power values." so maybe this is expected?

**Reply by Rem0o** [reply, marked as answer]
That would explain it. The "?" compare the given value to the readback value. If there's a lag, that would explain it.

**Reply by mhomorody** [reply]
Thanks, I guess it's working then. Great software, donation sent.
